tests/test_rep_file_parse.py

```diff
@@ -162,6 +163,58 @@ def test_line_ok(self):
         self.assertEqual(0.0, rep_line.depth)
         self.assertEqual("Label", rep_line.text_label)
 
+    def test_zero_mins_secs(self):
+        rep_line = REPLine(
+            line_number=1,
+            line=create_test_line_object(
+                "100112\t120800\tSUBJECT\tVC\t53.243\t0\t0\tS\t23.495\t00\t0\tE\t109.08\t6.00\t0.00\tLabel"
+            ),
+            separator="\t",
+        )
+        assert rep_line.parse(self.error, self.message)
+
+        correct_loc = Location()
+        correct_loc.set_latitude_decimal_degrees(-53.243)
+        correct_loc.set_longitude_decimal_degrees(23.495)
+        assert correct_loc == rep_line.location
+
+    def test_zero_secs(self):
+        rep_line = REPLine(
+            line_number=1,
+            line=create_test_line_object(
+                "100112\t120800\tSUBJECT\tVC\t53\t45.32\t0\tS\t23\t56.23\t0\tE\t109.08\t6.00\t0.00\tLabel"
+            ),
+            separator="\t",
+        )
+        assert rep_line.parse(self.error, self.message)
+
+        correct_loc = Location()
+        correct_loc.set_latitude_dms(53, 45.32, 0, "S")
+        correct_loc.set_longitude_dms(23, 56.23, 0, "E")
+        assert correct_loc == rep_line.location
+
+    def test_nan_depth(self):
+        rep_line = REPLine(
+            line_number=1,
+            line=create_test_line_object(
+                "100112\t120800\tSUBJECT\tVC\t60\t23\t40.25\tS\t000\t01\t25.86\tE\t109.08\t6.00\tNaN\tLabel"
+            ),
+            separator="\t",
+        )
+        assert rep_line.parse(self.error, self.message)
+
+        assert isnan(rep_line.depth)
+
+    def test_extended_symbology(self):
+        rep_line = REPLine(
+            line_number=1,
+            line=create_test_line_object(
+                "100112\t120800\tSUBJECT\t@C[SYMBOL=torpedo,LAYER=Support]\t60\t23\t40.25\tS\t000\t01\t25.86\tE\t109.08\t6.00\t0.0\tLabel"
+            ),
+            separator="\t",
+        )
+        assert rep_line.parse(self.error, self.message)
+
```

Comment:
@robintw  - do we need any additional asserts in here, just to check that, for example, the lat/long got loaded?

Reply:
Good idea. Fixed in 7d8d684